src/ReachSets/ContinuousPost/BFFPS19/reach_blocks.jl

```diff
@@ -0,0 +1,269 @@
+
+
+#helper functions
+@inline function deco_post_sparse(b, blocks, Whatk_blocks, partition,
+                                  ϕpowerk, Xhat0, output_function)
+    for i in 1:b
+        bi = partition[blocks[i]]
+        Xhatk = ZeroSet(length(bi))
+        for (j, bj) in enumerate(partition)
+          block = ϕpowerk[bi, bj]
+          if !iszero(block)
+              Xhatk = Xhatk + block * Xhat0[j]
+          end
+        end
+        Xhatk_lazy = (U == nothing ? Xhatk : Xhatk + Whatk_blocks[i])
+        Xhatk_d[i] = (output_function == nothing) ?
```

Comment:
`Xhatk_d` needs to become an argument of this function.

Reply:
Or I can just fix a typo and make it `Xhatk`.

Reply:
No, it was not a typo. `Xhatk` is a set above.

Reply:
Indeed, but it was still wrong naming. I fixed it now and checked that it is consistent with old algorithm.